workspace/TS100/Core/Inc/Settings.h

```diff
@@ -49,6 +49,8 @@ typedef struct {
 
 	uint8_t powerLimitEnable; // Allow toggling of power limit without changing value
 	uint8_t powerLimit;       // Maximum power iron allowed to output
+	
+	uint16_t TipThermoCoeff; // uV/C * 10
```

Comment:
If you feel like it, extending the comment with what it is as long with its unit would be nice :D